nim_mic_vad_streaming/linux_nim_vad_streaming/README.md

```diff
@@ -0,0 +1,70 @@
+# MICROPHONE VAD STREAMING
+Minimalistic example to demonstrate the DeepSpeech streaming  API in NIM.Raw audio is streamed from microphone to the DeepSpeech based on VAD (voice Activity Detection).
+
+## Prerequisites:
+0) Please read ``PREREQUISITES`` in [README](../README.md)  for getting the required ``libdeepspeech.so`` shared library.
+1) This example depends on the ``libasound.so``(which is distributed along with all major linux distros and present in linker's default path)
+
+_Note:  You may need to install ``libasound.so``  if not found_
+```
+sudo apt-get install libasound2
+```
+2) Download the pre-trained DeepSpeech english model (1089MB) and Scorer Package(~900MB):
+
+```
+wget https://github.com/mozilla/DeepSpeech/releases/download/v0.7.0/deepspeech-0.7.0-models.pbmm
+wget https://github.com/mozilla/DeepSpeech/releases/download/v0.7.0/deepspeech-0.7.0-models.scorer
+```
+
+
+## Installation
+
+1. Install Nim bindings for deespeech version-0.7.0 .
+```nim
+nimble install https://gitlab.com/eagledot/nim-deepspeech@0.7.0
```

Comment:
If you have bindings, please make a PR on https://github.com/mozilla/DeepSpeech to document them